src/components/data_search/DatasetSearchTableConstants.tsx

```diff
@@ -243,7 +243,7 @@ export const makeDatasetTableHeader = (datasets: DatasetTerm[], selected: number
     duosId: '10%',
     accessType: '10%',
     dataType: '15%',
-    donorSize: '7%',
+    donorSize: '10%',
```

Comment:
Before the total %s equalled 100 when added up (25 + 10 + 10+ 15 + 7 + 13  + 10). Now it's 103%. Is that correct? If this % goes up, maybe another needs to go down?

Reply:
I thought that too, but it doesn't seem to cause any weird behavior in the rendering of the table itself, but I'll look more into it

Reply:
Widths can be > 100%. It can sometimes lead to overflow issues.